cogs/deadline.py

```diff
@@ -197,6 +210,10 @@ async def deleteReminder_error(self, ctx, error):
                       help="update the assignment date. $changeduedate CLASSNAME HW_NAME MMM DD YYYY optional(HH:MM) "
                       "ex. $changeduedate CSC510 HW2 SEP 25 2024 17:02 ")
     async def changeduedate(self, ctx, classid: str, hwid: str, *, date: str):
+        if not ctx.channel.name == 'assignments':
+            await ctx.author.send('Please send questions to the #assignments channel.')
```

Comment:
'Please send questions to the #assignments channel.'

Should be:

'Please use this command inside the #assignments channel.'